test/Microsoft.Sbom.Tool.Tests/IntegrationTests.cs

```diff
@@ -226,18 +248,15 @@ public void E2E_Redact_WithManifestInfo_ReturnsNonZeroExitCode(string manifestIn
         Assert.AreNotEqual(0, exitCode.Value);
     }
 
-    private void GenerateManifestAndValidateSuccess(string testFolderPath, string manifestInfoValue = null)
+    private void GenerateManifestAndValidateSuccess(string testFolderPath, string manifestInfoSpdxVersion = null)
     {
         var arguments = $"generate -ps IntegrationTests -pn IntegrationTests -pv 1.2.3 -m \"{testFolderPath}\"  -b \"{testDropDirectory}\" -bc \"{GetSolutionFolderPath()}\"";
-        if (!string.IsNullOrEmpty(manifestInfoValue))
-        {
-            arguments += $" -mi SPDX:{manifestInfoValue}";
-        }
+        arguments += !string.IsNullOrEmpty(manifestInfoSpdxVersion) ? $" -mi SPDX:{manifestInfoSpdxVersion}" : string.Empty;
```

Comment:
This is a funny mix of interpolation and concatenation. How would you feel about making it all concatenation? One option:
```
var manifestInfoArg = string.IsNullOrEmpty(manifestInfoSpdxVersion) ? string.Empty : $"-mi SPDX:{manifestInfoSpdxVersion}" ;
var arguments = $"generate -ps IntegrationTests -pn IntegrationTests -pv 1.2.3 -m \"{testFolderPath}\" -b \"{testDropDirectory}\" -bc \"{GetSolutionFolderPath()}\" {manifestInfoArg}";
```

Reply:
Makes sense to me, I made the change.